_layouts/home.html

```diff
@@ -1,78 +1,93 @@
 ---
 layout: default
 ---
+<div id="home">
+{% include header.html %}
 
+<div class="header-image">
```

Comment:
here's the issue to track this: https://github.com/daiyi/mypaint.github.io/issues/7